### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by javaDeveloperKid**
I find the current comment confusing.
1. `skip` in comment and `IMPORT...`  in constant name do not fit here together. Btw. `AUTO_IMPORT_NAMES` constant name is not clear to me. What "names"? It doesn't have to be changed but let's improve the comment here.
2. added very verbose and common term i.e. `... use statements` , because `import` word in context of what Rector does here can also be understood as `add type hint for root namespace classes`  
3. I don't know IMPORT_DOC_BLOCKS option yet, but I may require the same reword. I will have a look at this later or you can propose you own based on my 2 rewords.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,10 +166,16 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     // is your PHP version different from the one your refactor to? [default: your PHP version], uses PHP_VERSION_ID format
     $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_72);
 
-    // auto import fully qualified class names? [default: false]
+    // whether Rector can leave FQCN or automatically add them to use statements and remove from code occurences [default: false]
```
Comment (TomasVotruba) [inline README.md:169]: A or B option with true or false,  is like "do you have pizza or cake", yes or no.
Better leave one option.

File: README.md [hunk 2]
```diff
@@ -166,10 +166,16 @@ return static function (ContainerConfigurator $containerConfigurator): void {
...
+    // e.g. 
+    // private \My\Namespace\MyService $field;
+    // vs
+    // use My\Namespace\MyService;
+    // ...
+    // private MyService $field;
```
Comment (TomasVotruba) [inline README.md:175]: This is confusing. Which is before after? Why partial?
One clear diff example would be better.

File: README.md (hunk 1 above)
Comment (javaDeveloperKid) [reply README.md:169]: I'm affraid I don't understand :)

Comment (TomasVotruba) [reply README.md:169]: There is a binary option with binary answer: `// whether Rector can A or B`

Comment (javaDeveloperKid) [reply README.md:169]: I think I know what's your point. I used `or` an then `and`. i think a comma would be helpful or as we are developers some parentheses what to read first :)

**Comment by TomasVotruba** [comment]
I think this would deserve own page in the docs, where this would be linked and explained better.
There 3 options get sometimes confused, so it would be nice to have dedicated source that we can link next time.

**Comment by javaDeveloperKid** [comment]
Ok, I will give it a try this week and we will figure out the best description.

**Comment by TomasVotruba** [comment]
Closing to avoid staling, with this output https://github.com/rectorphp/rector/pull/5754#issuecomment-789713612

Feel free to re-open with PR that adds new docs file :+1:

**Pull request closed without merging**